content/docs/next-release/apis.md

```diff
@@ -8,7 +8,7 @@ Jaeger components implement various APIs for saving or retrieving trace data.
 The following labels are used to describe API compatibility guarantees.
 
 * **stable** - the API guarantees backwards compatibility. If breaking changes are going to be made in the future, they will result in a new API version, e.g. `/api/v2` URL prefix or a different namespace in the IDL.
-* **internal** - the APIs intended for internal communications between Jaeger components and not recommended for use by external components.
+* **internal** - the APIs intended for internal communications between Jaeger components and are not recommended for use by external components.
```

Comment:
`not recommended` acts as an adjective here, same as `intended` earlier. If you want to turn these into verbs then it should also say `are intended` (which will match the structure of 1st and 3rd bullets)
```suggestion
* **internal** - the APIs are intended for internal communications between Jaeger components and are not recommended for use by external components.
```